src/main/resources/org/craftercms/studio/api/v2/dal/ItemDAO.xml

```diff
@@ -580,4 +580,22 @@
         AND s.deleted = 0
         AND i.path = #{path}
     </select>
+
+    <select id="getContentTypeUsages" resultType="org.craftercms.studio.api.v2.dal.Item">
+        SELECT
+          I.path, I.system_type AS systemType
+        FROM
+          item I
+        LEFT JOIN
```

Comment:
Switch to INNER JOIN

Reply:
Get intersection of two tables on the site_id, and THEN filter by site and type.